source/extensions/filters/http/proto_api_scrubber/filter.h

```diff
@@ -39,6 +44,12 @@ class ProtoApiScrubberFilter : public Http::PassThroughFilter,
   // Request message converter which converts Envoy Buffer data to StreamMessage (for scrubbing) and
   // vice-versa.
   GrpcFieldExtraction::MessageConverterPtr request_msg_converter_{nullptr};
+
+  // An instance of ProtoScrubber for scrubbing request protobuf payloads.
+  std::unique_ptr<ProtoScrubber> request_scrubber_{nullptr};
+
+  // Field checker for request scrubbing.
+  std::unique_ptr<FieldChecker> request_field_checker_{nullptr};
```

Comment:
Two questions:
- why is this a pointer? isn't a normal field sufficient?
- if understand the code correctly, the `request_scrubber_` is using the `request_field_checker_`. Will it make sense to move `request_field_checker_` before the other field (just to make sure that the `request_scrubber_` is destroyed properly before the field checker)?